controller/relay-image.go

```diff
@@ -24,26 +25,27 @@ func relayImageHelper(c *gin.Context, relayMode int) *OpenAIErrorWithStatusCode
 	group := c.GetString("group")
 
 	var imageRequest ImageRequest
+
 	if consumeQuota {
 		err := common.UnmarshalBodyReusable(c, &imageRequest)
 		if err != nil {
 			return errorWrapper(err, "bind_request_body_failed", http.StatusBadRequest)
 		}
 	}
 
-	// Prompt validation
-	if imageRequest.Prompt == "" {
-		return errorWrapper(errors.New("prompt is required"), "required_field_missing", http.StatusBadRequest)
+	// Model validation
+	if imageRequest.Model != "" || imageRequest.Model != "dall-e-3" {
```

Comment:
By using the `||` operator, it will always be `true` here, and the `&&` operator should be used instead. 

```go
imageModel := imageRequest.Model
if imageRequest.Model != "" && imageRequest.Model != "dall-e-3" {
	imageModel = "dall-e-2"
}
```

Additionally, instead of defaulting to using a model here, it can throw an error.

~~`if imageRequest.Model != "" && imageRequest.Model != "dall-e-3" {  ....`~~

Reply:
I made changes in my local version but I forgot to push lol